### ISSUE TITLE
Panic when closing application with 0.10.3

### ISSUE THREAD

**Issue by dheijl** [MoAlyousef/fltk-rs#327]
> thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', C:\Users\Danny\.cargo\registry\src\github.com-1ecc6299db9ec823\fltk-0.10.3\src\app.rs:967:40
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

To reproduce: compile and run swyh-rs (https://github.com/dheijl/swyh-rs/), let it discover devices and capture audio and then close the application with the x button in the right top corner.

It's entirely possible that it is my own fault but I didn't change anything except for the fltk-rs version.

**Comment by MoAlyousef** [comment]
I was able to replicate the issue on swyh-rs. I'm guessing the std::process::exit is triggered before fltk-rs has the chance to do its cleanup asynchronously since it does it in the next draw cycle. Anyways I have removed the unwrapping (which I had assumed before would never happen), and the problem seems to have been resolved. 
The changes are in master, I'll push a patch 0.10.4 in a few hours when I'm on my main machine.
